### PR TITLE
Hotfix/1131 vehicle localization

### PR DESCRIPTION
**Pull request by Hoodathunk**
Added vehicle (SLF & SRV) definition and loadout description localization.
Corrects #1331 FDEV cAPI bug.

### CODE REVIEW

File: DataDefinitions/Vehicle.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ public static Vehicle FromJson(int subslot, dynamic json)
         {
             if (json is null) { return null; }
 
-            string edName = (string)json["EDName"];
+            string edName = (string)json["name"];
```
Comment (richardbuckle) [inline DataDefinitions/Vehicle.cs]: Are you 100% sure about this and the related changes to the test data? I was definitely seeing 'EDName' in the json.

Comment (Hoodathunk) [reply DataDefinitions/Vehicle.cs]: Yes, this is correct. Part of json from my Python for laucnhbays...
```    Slot09_Size4: {
        SubSlot0: {
            name: "testbuggy",
            locName: "SRV Scarab",
            rebuilds: 1,
            loadout: "starter",
            loadoutName: "Starter"
        },
        SubSlot1: {
            name: "testbuggy",
            locName: "SRV Scarab",
            rebuilds: 1,
            loadout: "starter",
            loadoutName: "Starter"
        }
    }
}
```

Comment (richardbuckle) [reply DataDefinitions/Vehicle.cs]: I guess I must have been looking at old JSON then. Le sigh.

Comment (Tkael) [reply DataDefinitions/Vehicle.cs]: I can confirm @Hoodathunk's json format.

File: Tests/DataDefinitionTests.cs [hunk 2]
```diff
@@ -186,5 +189,21 @@ public void TestAddOrUpdateBodyUpdate()
             Assert.AreEqual("testSystem 1", actualBody.bodyname);
             Assert.AreEqual(moon, actualBody.bodyType);
         }
+
+        [TestMethod]
+        [DeploymentItem("vehicle.json")]
```
Comment (Tkael) [inline Tests/DataDefinitionTests.cs]: This may not be 100% compatible with my code coverage branch (which seems to fail whenever presented with a test with a deployment item). Consequently, the code coverage branch embeds the resources into Tests.dll where code coverage can be happily evaluated. 
TLDR: Some post-merge patching may be required if both PRs are accepted. 🤷‍♂
